pegasus/sites.v3/hourofcode.com/views/whats_next.haml

```diff
@@ -3,4 +3,4 @@
   %ul{:style=>"line-height:28px"}
     %li=hoc_s(:whats_next_how_to).gsub('/how-to', resolve_url('/how-to'))
     %li=hoc_s(:whats_next_how_to_promote_link).gsub('/promote', resolve_url('/promote'))
-    %li=hoc_s(:whats_next_try).gsub('https://code.org/learn', resolve_url('https://code.org/learn'))
+    %li=hoc_s(:whats_next_try).gsub('%{learn_url}', resolve_url('/learn'))
```

Comment:
Presumably this means that on hourofcode.com a translator can put arbitrary HTML into a translation string, and so we have the risk that @Bjvanminnen pointed out in the recent tutorialExplorer PR.  Understand that this is status quo, just wanted to confirm.